lua/pqf/init.lua

```diff
@@ -285,6 +290,9 @@ function M.setup(options)
         'max_filename_length must be a number'
       )
     end
+    if options.hide_placeholder_path then
+        hide_placeholder_path = true
+    end
```

Comment:
Could you run [stylua](https://github.com/JohnnyMorganz/StyLua) on this file? It seems the indentation isn't consistent.

Reply:
my bad, I'm using 4 spaces and didn't see it